joplin/base/wagtail_hooks.py

```diff
@@ -58,7 +58,7 @@ def configure_main_menu(request, menu_items):
     menu_items[:] = [item for item in menu_items if item.name not in
                      # here were excluding some default generated menu items per UX
                      [
-                         'explorer',
+                         # 'explorer',
```

Comment:
we can re-hide this later, atm I'm using it as an interface to show how index pages might work. in another branch i have an example using ModelAdmin, but thats probably overkill for this situation